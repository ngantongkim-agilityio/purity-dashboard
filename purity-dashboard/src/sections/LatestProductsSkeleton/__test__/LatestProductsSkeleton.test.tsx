import { render } from '@testing-library/react';

// Components
import { LatestProductsSkeleton } from '..';

describe('LatestProductsSkeleton component', () => {
  it('Should render LatestProductsSkeleton component with default props', () => {
    const { container } = render(<LatestProductsSkeleton />);

    expect(container).toMatchSnapshot();
  });
});
