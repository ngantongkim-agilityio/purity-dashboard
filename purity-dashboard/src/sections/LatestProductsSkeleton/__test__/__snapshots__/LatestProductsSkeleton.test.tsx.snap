// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LatestProductsSkeleton component Should render LatestProductsSkeleton component with default props 1`] = `
<div>
  <div
    class="before:absolute before:inset-0 before:-translate-x-full before:animate-[shimmer_2s_infinite] before:bg-gradient-to-r before:from-transparent before:via-white/60 before:to-transparent relative flex w-full flex-col overflow-hidden md:col-span-4"
  >
    <div
      class="mb-4 h-8 w-36 rounded-md bg-gray-100"
    />
    <div
      class="flex grow flex-col justify-between rounded-xl bg-gray-100 p-4"
    >
      <div
        class="bg-white px-6"
      >
        <div
          class="flex flex-row items-center justify-between border-b border-gray-100 py-4"
        >
          <div
            class="flex items-center"
          >
            <div
              class="mr-2 h-8 w-8 rounded-full bg-gray-200"
            />
            <div
              class="min-w-0"
            >
              <div
                class="h-5 w-40 rounded-md bg-gray-200"
              />
              <div
                class="mt-2 h-4 w-12 rounded-md bg-gray-200"
              />
            </div>
          </div>
          <div
            class="mt-2 h-4 w-12 rounded-md bg-gray-200"
          />
        </div>
        <div
          class="flex flex-row items-center justify-between border-b border-gray-100 py-4"
        >
          <div
            class="flex items-center"
          >
            <div
              class="mr-2 h-8 w-8 rounded-full bg-gray-200"
            />
            <div
              class="min-w-0"
            >
              <div
                class="h-5 w-40 rounded-md bg-gray-200"
              />
              <div
                class="mt-2 h-4 w-12 rounded-md bg-gray-200"
              />
            </div>
          </div>
          <div
            class="mt-2 h-4 w-12 rounded-md bg-gray-200"
          />
        </div>
        <div
          class="flex flex-row items-center justify-between border-b border-gray-100 py-4"
        >
          <div
            class="flex items-center"
          >
            <div
              class="mr-2 h-8 w-8 rounded-full bg-gray-200"
            />
            <div
              class="min-w-0"
            >
              <div
                class="h-5 w-40 rounded-md bg-gray-200"
              />
              <div
                class="mt-2 h-4 w-12 rounded-md bg-gray-200"
              />
            </div>
          </div>
          <div
            class="mt-2 h-4 w-12 rounded-md bg-gray-200"
          />
        </div>
        <div
          class="flex flex-row items-center justify-between border-b border-gray-100 py-4"
        >
          <div
            class="flex items-center"
          >
            <div
              class="mr-2 h-8 w-8 rounded-full bg-gray-200"
            />
            <div
              class="min-w-0"
            >
              <div
                class="h-5 w-40 rounded-md bg-gray-200"
              />
              <div
                class="mt-2 h-4 w-12 rounded-md bg-gray-200"
              />
            </div>
          </div>
          <div
            class="mt-2 h-4 w-12 rounded-md bg-gray-200"
          />
        </div>
        <div
          class="flex flex-row items-center justify-between border-b border-gray-100 py-4"
        >
          <div
            class="flex items-center"
          >
            <div
              class="mr-2 h-8 w-8 rounded-full bg-gray-200"
            />
            <div
              class="min-w-0"
            >
              <div
                class="h-5 w-40 rounded-md bg-gray-200"
              />
              <div
                class="mt-2 h-4 w-12 rounded-md bg-gray-200"
              />
            </div>
          </div>
          <div
            class="mt-2 h-4 w-12 rounded-md bg-gray-200"
          />
        </div>
      </div>
      <div
        class="flex items-center pb-2 pt-6"
      >
        <div
          class="h-5 w-5 rounded-full bg-gray-200"
        />
        <div
          class="ml-2 h-4 w-20 rounded-md bg-gray-200"
        />
      </div>
    </div>
  </div>
</div>
`;
